### ISSUE TITLE
Anyone has been able to get this running recently on Mac

### ISSUE THREAD

**Issue by adam-lokicode** [slockit/piet#45]
Hi, if anyone could provide an updated guide on how to run this on Mac that would be great.  Just encountering countless problems trying to get this to run.

**Comment by thisaldesilva** [comment]
Yes, same. I could not get it run. npm i halt operations with the following : 503 Service Unavailable - GET https://npm.slock.it/yauzl/-/yauzl-2.10.0.tgz
